src/overlays/ovl_i4/fox_fo.c

```diff
@@ -1,29 +1,1513 @@
+            Math_SmoothStepToF(&player->unk_0E8, -120.0f, 0.1f, 2.0f, 0.0f);
+            player->unk_0D0 += 1.0f;
+            if (player->unk_0D0 >= 70.0f) {
+                player->unk_0D0 = 70.0f;
+                player->unk_25C += 0.04f;
+                if (player->unk_25C > 0.6f) {
+                    player->unk_25C = 0.6f;
+                }
+            }
+            if (player->timer_1F8 == 0) {
+                player->unk_1D0 = 2;
+                player->timer_1F8 = 1000;
+                for (i = 0; i < ARRAY_COUNT(gActors); i++) {
+                    Object_Kill(&gActors[i].obj, gActors[i].sfxPos);
+                }
+                //! FAKE:
+                if (((&dest) && (&dest)) && (&dest)) {}
+
+                func_8001CA24(0U);
+                Audio_KillSfx(player->sfxPos);
+                player->unk_234 = 0;
+                D_80178340 = 255;
+                D_80178358 = 255;
+                D_80178348 = D_80178350 = D_80178354 = 0;
+                D_8017835C = 0;
+            }
+            break;
+
+        case 2:
+            if (!(D_80177930) && (player->timer_1F8) > 830) {
+                D_80178358 = 0;
+                D_8017835C = 8;
+            }
+            if (player->timer_1F8 == 810) {
+                Audio_KillSfx(gBosses[0].sfxPos);
+            }
+            if (player->timer_1F8 == 830) {
+                D_80178358 = 255;
+                D_8017835C = 2;
+                D_80178348 = D_80178350 = D_80178354 = 255;
+            }
+
+            gBosses[0].swork[0] = 1;
+
+            if ((func_80090200(gBosses) == 2) || (D_80177930 != 0)) {
+                func_800A6148();
+                if (D_80177930 == 0) {
+                    player->unk_1D0 = 10;
+                } else {
+                    player->unk_1D0 = 20;
+                }
+
+                for (i = 0; i < 200; i++) {
+                    gObjects58[i].obj.status = 0;
+                }
+
+                func_800A5EBC();
+                gLevelType = 1;
+                D_801784AC = gBgColor = gFogRed = gFogGreen = gFogBlue = 0;
+                gLight1R = gLight2R = D_80161A70 = 86;
+                gLight1G = gLight2G = D_80161A74 = 58;
+                gLight1B = gLight2B = D_80161A78 = 25;
+                gAmbientR = 11;
+                gAmbientG = 8;
+                gAmbientB = 24;
+                D_801784C4 = -59.0f;
+                D_801784F8 = D_801784C4;
+                D_801784C4 = D_801784F8;
+                D_801784C4 = D_801784C4;
+                D_801784D0 = D_801784C4;
+                D_801784C8 = 58.0f;
+                D_801784FC = D_801784C8;
+                D_801784C8 = D_801784FC;
+                D_801784C8 = D_801784C8;
+                D_801784D4 = D_801784C8;
+                D_801784CC = 13.0f;
+                D_80178500 = D_801784CC;
+                D_801784CC = D_80178500;
+                D_801784CC = D_801784CC;
+                D_801784D8 = D_801784CC;
```

Comment:
```suggestion
                D_801784D0 = D_801784C4 = D_801784C4 = D_801784F8 = D_801784C4 = -59.0f;
                D_801784D4 = D_801784C8 = D_801784C8 = D_801784FC = D_801784C8 = 58.0f;
                D_801784D8 = D_801784CC = D_801784CC = D_80178500 = D_801784CC = 13.0f;
```

I think these are likely chain assigns. Should reduce the padding needed for the stack too